alembic/versions/0e105f6e36b4_add_a_commit_column_to_the_builds_table.py

```diff
@@ -0,0 +1,25 @@
+"""Add a commit column to the builds table.
+
+Revision ID: 0e105f6e36b4
+Revises: c6a5e2849ca4
+Create Date: 2017-05-11 18:35:40.550295
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0e105f6e36b4'
+down_revision = 'c6a5e2849ca4'
+
+
+def upgrade():
+    """Add a commit column to the builds table with a uniqueness constraint."""
+    op.add_column('builds', sa.Column('commit', sa.Unicode(length=100), nullable=True))
```

Comment:
OK, so the longest project name we have with a current git hash would be 127 characters:

```
>>> len("git://pkgs.fedoraproject.org/rpms/golang-github-matttproud-golang_protobuf_extensions?#34e14931464563b7d549816d1a2324c9af444d8e")
127
```

If git switches to a sha512 later, that same long name would be 215 characters:

```
>>> len("git://pkgs.fedoraproject.org/rpms/golang-github-matttproud-golang_protobuf_extensions?#e7c22b994c59d9cf2b48e549b1e24666636045930d3da7c1acb299d1c3b7f931f94aae41edda2c2b207a36e10f8bcb8d45223e54878f5b316e7ce3b6bc019629")
215
```

I propose we use 256 characters since that's a pleasing power of 2 and gives even a little more room for growth? Or we could just use 150 or so and bump it as needed, but I say why not just go ahead and make room for sha512.